canonical-web-and-design/snapcraft.io
# Image template

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/2291

## QA
- Pull the branch
- `./run`
- Visit http://0.0.0.0:8004
- Visit http://0.0.0.0:8004/store
Things should load quicker
